src/schema/types/user.graphql

```diff
@@ -5,4 +5,19 @@ type User {
     username: String!
     salt: String!
     password: String!
-}
\ No newline at end of file
+}
+extend type Query {
+  users: [User!]!,
+  user(id: ID!): User
+}
+extend type Mutation {
+  createUser(
```

Comment:
It's a good practice to create an `Input` type, instead of passing many parameters to the mutation.

Eg:
```graphql
createUser(user: UserCreationInput): User!
```